### ISSUE TITLE
IBR - Change the effects in the focus "A close relationship"

### ISSUE THREAD

**Issue by Marok132** [TNODevs/TNO-Bug-Tracker#3279]
### HOI4 version

1.14.5

### TNO version

1.5.0c

### Which major expansions do you NOT have?

I have all

### Describe your suggestion

To change the effects in the focus "A close relationship" from 50 pps to getting pakt observer/full membership or other effects that are better than 50 pps.

### Justifications

1. It's a final focus in the pro-German diplomacy focus tree, and it should give something more benefits than just 50 pps. It feels like trolling on someone who wanna join the Pakt as the Iberian Union but gets dissapointed when they realize they get ludicrous effects instead of.
2. Just like the last focus in the pro-American diplomacy tree where you join the OFN (or get OFN observer as Franco), the last focus in the German one should do the same.
3. The narrative of the focus tree is supposed you will get more closer connections with the Pakt and Germany (inviting the corporation, investing in Eastern Europe. cooperation between intelligence, and so on). So I think getting  the Pakt observer is a obvious end of type of relationship.

### Screenshots

![image](https://github.com/TNODevs/TNO-Bug-Tracker/assets/161346330/d258b7ae-8d46-4d61-bfa6-2b57ef32aa75)
![image](https://github.com/TNODevs/TNO-Bug-Tracker/assets/161346330/eb57aef6-0da2-40d2-8d77-14307ddd0f77)

**Comment by Marok132** [comment]
@ModdingAccount @Stormeagle449 @Jack121Q Could this suggestion be reviewed, please?

**Comment by Jack121Q** [comment]
I don't believe Iberia is interested in that. If Speer is in charge, they join the Zollverein with the focus though.
